package.json

```diff
@@ -11,7 +11,9 @@
     "url": "git://github.com/thlorenz/convert-source-map.git"
   },
   "homepage": "https://github.com/thlorenz/convert-source-map",
-  "dependencies": {},
+  "dependencies": {
+    "safe-buffer": "^5.1.1"
```

Comment:
change `^` to `~` to protect from breakage (I understand semver but don't trust libraries I don't control to understand it as well)